plugins/hcp/commands/cluster-health-check.md

```
@@ -0,0 +1,712 @@
+---
+description: Perform comprehensive health check on HCP cluster and report issues
+argument-hint: "[--verbose] [--output-format]"
+---
+
+## Name
+hcp:cluster-health-check
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)